### PR TITLE
Update job example with new console screenshots

### PR DESCRIPTION
**Pull request by shpface**
*Issue #, if available:*

*Description of changes:*

This change will update the screenshots in the example demonstrating creation of a hybrid job from the Braket console.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: examples/hybrid_jobs/8_Creating_Hybrid_Job_Scripts/Creating_your_first_Hybrid_Job.ipynb [hunk 1]
```diff
@@ -526,39 +526,47 @@
    "source": [
```
Comment (ashlhans) [inline examples/hybrid_jobs/8_Creating_Hybrid_Job_Scripts/Creating_your_first_Hybrid_Job.ipynb:526]: <p>...The script can be uploaded directly <strong>from your computer</strong>...</p><p> </p><p>...After the algorithm is specified, you <strong>have the option</strong> to skip directly to the review and create page...</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/amazon-braket/amazon-braket-examples/pull/545/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>

Comment (ashlhans) [inline examples/hybrid_jobs/8_Creating_Hybrid_Job_Scripts/Creating_your_first_Hybrid_Job.ipynb:526]: <p>Next, you can choose a name for your hybrid job<strong> or use the</strong> default name.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/amazon-braket/amazon-braket-examples/pull/545/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>

Comment (ashlhans) [inline examples/hybrid_jobs/8_Creating_Hybrid_Job_Scripts/Creating_your_first_Hybrid_Job.ipynb:526]: <p>"Pennylane and the BYOC example notebooks." Could you link to an example here?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/amazon-braket/amazon-braket-examples/pull/545/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>

Comment (ashlhans) [inline examples/hybrid_jobs/8_Creating_Hybrid_Job_Scripts/Creating_your_first_Hybrid_Job.ipynb:526]: <p>Could you link to the other examples with the advanced use cases?</p><p> </p><p>Nit:"You may" is used at the beginning of both sentences. Maybe "You also have the option to customize the default locations for checkpoints and output data." would read better</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/amazon-braket/amazon-braket-examples/pull/545/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>

**Review by ashlhans — APPROVED** [review]

**Comment by krneta** [comment]
Why are we doing this in a notebook? This seems like it belongs in a blog post or documentation, but not if customers already have a programmatic interface (as they do in these examples). Maybe we can just point them to the docs?

**Pull request merged**